static/js/components/dashboard/CourseStatus.js

```diff
@@ -1,50 +1,28 @@
 // @flow
```

Comment:
FYI: all of this code is essentially deprecated. we're no longer showing the price in the center column of the course row, and it didn't make sense to refactor this to update the handling of the course price. there is another story to show the current grade instead in this column